sdk/trace/provider.go

```diff
@@ -236,13 +257,17 @@ func (p *TracerProvider) ForceFlush(ctx context.Context) error {
 
 // Shutdown shuts down TracerProvider. All registered span processors are shut down
 // in the order they were registered and any held computational resources are released.
+// After Shutdown is called, all other methods are no-ops.
```

Comment:
this one is no-op as well 😄 
```suggestion
// After Shutdown is called, all methods are no-ops.
```